### ISSUE TITLE
Add pipeline

### ISSUE THREAD

**Issue by crapStone** [Calciumdibromid/CaBr2#9]
Add a pipeline to run angular and rust tests.

**Comment by 6543** [comment]
I would call it CI ... yes we should add it.

I would take this, but it depends on adding a nice test suite ...

so blocked by #8 ?

**Comment by 6543** [comment]
close as duplicate of #8
